### PR TITLE
[tests] Add more tests for `importBuilders()` scenarios

### PR DESCRIPTION
**Pull request by TooTallNate**
Covers:
  - Importing a built-in Builder with `@latest` should use the CLI's version
  - Importing a built-in Builder with `@canary` should use the CLI's version
  - Importing a built-in Builder with an explicit version should install to `.vercel/builders`

### CODE REVIEW

File: packages/cli/test/unit/util/build/import-builders.test.ts [hunk 1]
```diff
@@ -27,7 +27,67 @@ describe('importBuilders()', () => {
...
+    expect(builders.get('@vercel/next@latest')?.pkg).toMatchObject(
+      vercelNextPkg
+    );
+    expect(typeof builders.get('@vercel/node@latest')?.builder.build).toEqual(
+      'function'
+    );
+    expect(typeof builders.get('@vercel/next@latest')?.builder.build).toEqual(
+      'function'
+    );
+  });
```
Comment (styfle) [inline packages/cli/test/unit/util/build/import-builders.test.ts:60]: Can we verify that no "npm install" was run?

Perhaps check `client.stderr`?

```js
expect(client.stderr).not.toMatch('Installing Builder');
```

Comment (TooTallNate) [reply packages/cli/test/unit/util/build/import-builders.test.ts:60]: The "not" case depends on a timeout so I prefer not to use that if necessary. I'd rather check the imported file path. Will add a case for that.

**Review by cb1kenobi — APPROVED** [review]

File: packages/cli/test/unit/util/build/import-builders.test.ts (hunk 1 above)
Comment (TooTallNate) [reply packages/cli/test/unit/util/build/import-builders.test.ts:60]: dc7d3a6b30c3a8d24ca4d8f91c6778f36d0d4877

**Review by styfle — APPROVED** [review]
Great, thanks!

**Pull request merged**